README.md

```diff
@@ -1,137 +1,211 @@
 # Awesome OSS Management [![Awesome](https://awesome.re/badge-flat.svg)](https://awesome.re)
 
-This list identifies packages and projects that have been built by TODO Group
-members or found helpful for managing open source projects and offices.
-
-## Contents
-* [Code Reviews](#code-reviews)
-* [Continuous Integration / Continuous Delivery](#continuous-integration--continuous-delivery)
-* [Contributor License Agreements / Developer Certificate of Origins](#contributor-license-agreements--developer-certificate-of-originis)
-* [GitHub Metrics and Dashboards](#github-metrics-and-dashboards)
-* [GitHub Management](#github-management)
-* [Project Quality](#project-quality)
-* [Supply Chain Trust](#supply-chain-trust)
-* [Licensing](#licensing)
-* [Localization and Internationalization](#localization-and-internationalization)
-* [Websites and Documentation](#websites-and-documentation)
-* [Security](#security)
-* [In-Kind Donations](#in-kind-donations)
-* [Content License](#content-license)
-
-## Code Reviews
+This list identifies packages and projects that have been built by [TODO Group
+members](https://github.com/todogroup/awesome-ospo) and [Ant Group OSPO](https://github.com/antgroup/) or found helpful for managing open source projects and offices.
```

Comment:
Please remove the mention of Ant Group here.
While we have several contributors to this repo, we do not currently list them individually (and so it would be inappropriate to list just one in this PR).